test/functional/users_controller_test.rb

```diff
@@ -288,4 +288,10 @@ def setup
     assert_response :redirect
     assert_equal I18n.t('users_controller.no_user_found'), flash[:error]
   end
+  test 'should assign correct value to noteGraph' do
```

Comment:
Then with the above change to simplify the controller, we can actually test `user.user.daily_note_tally.to_a.sort` in the unit tests for `user` -- and let's do some basic things like assert it's not zero, assert that it has the right array length, assert that the array members are integers, things like that. 

Then here, we could do an `assert_select` for the graph itself and perhaps also scan the body for the array? But only if we want to be super thorough. 

Thank you!!!